### ISSUE TITLE
Catalog Categories not updating properly with REST API

### ISSUE THREAD

**Issue by allamsettiramesh** [magento/devdocs#1284]
Team. I am trying to update category name, url-key, url-path with REST API and with following parameters but i can see updated "name" in front end & not in admin panel as well url-key & url-path also not updating 

Request Parameters:
   
  `{
  "category": {
    "id":22,
    "parentId": 2,
    "name": "Bags",
    "isActive": true,
    "position": 13,
    "level": 2,
    "children": "",
    "created_at": "2017-07-10 06:35:53",
    "updated_at": "2017-07-10 06:35:54",
    "path": "1/2/22",
    "available_sort_by": [],
    "include_in_menu": true,
    "customAttributes": [
         {
            "attribute_code": "is_anchor",
            "value": "1"
        },
        {
            "attribute_code": "path",
            "value": "1/2/22"
        },
        {
            "attribute_code": "children_count",
            "value": "0"
        },
        {
            "attribute_code": "url_key",
            "value": "bags"
        },
        {
            "attribute_code": "url_path",
            "value": "bags"
        }
    ]
  }
}`

Response:

  `{
    "id": 22,
    "parent_id": 2,
    "name": "Bags",
    "is_active": true,
    "position": 13,
    "level": 2,
    "children": "",
    "created_at": "2017-07-10 06:35:53",
    "updated_at": "2017-07-10 06:35:54",
    "path": "1/2/22",
    "available_sort_by": [],
    "include_in_menu": true,
    "custom_attributes": [
        {
            "attribute_code": "is_anchor",
            "value": "1"
        },
        {
            "attribute_code": "path",
            "value": "1/2/22"
        },
        {
            "attribute_code": "children_count",
            "value": "0"
        },
        {
            "attribute_code": "url_key",
            "value": "bags"
        },
        {
            "attribute_code": "url_path",
            "value": "bags"
        }
    ]
}`

**Comment by allamsettiramesh** [comment]
Can any one help on this

**Comment by keharper** [comment]
Hi @allamsettiramesh I can reproduce your problem on a 2.1.3 build I keep around, but everything works as expected on 2.2. Let me know what version you're working on, and I'll see if I can determine if the issue has been fixed in a subsequent 2.1 patch.

**Comment by allamsettiramesh** [comment]
@keharper, I am working on 2.1.6. version.

**Comment by keharper** [comment]
Bad news, @allamsettiramesh . The issue is still present in the upcoming 2.1.8 patch. I tried looking through the issues fixed for 2.2 to determine if this issue is scheduled to be backported, but I can't determine which ticket corrected the problem. This issue might have been fixed accidentally.

However, I've had some limited success specifying a specific store ID, rather than the global ID of `0`. The REST call returned all the expected values with storeId=1. In the admin panel. when you switch from All Store Views to Default Store View, the name and URL Key display correctly, but not the URL Rewrites.
